tensorflow_lite_support/python/task/vision/pybinds/image_classifier.cc

```diff
@@ -29,7 +29,7 @@ namespace {
 namespace py = ::pybind11;
 }  // namespace
 
-PYBIND11_MODULE(_pywrap_image_classifier, m) {
+PYBIND11_MODULE(image_classifier, m) {
```

Comment:
@ziyeqinghan I don't know why pybind works in OSS if being setup like this (set module name to `image_classifier`), but failed when module name is set as `_pywrap_image_classifier`.

You can run `bazel run //tensorflow_lite_support/python/test/task/vision:image_embedder_test` to reproduce that this test failed in OSS.

Reply:
Please change the file name `image_classifier.cc` to `_pywrap_image_classifier.cc` which should fix this.